src/applications/simple-forms/26-4555/config/form.js

```diff
@@ -77,7 +77,7 @@ const formConfig = {
         },
         personalInformation2: {
           path: 'personal-information-2',
-          title: "Personal Information (cont'd)",
+          title: "Identification Information",
```

Comment:
@jeana-adhoc Are the titles supposed to be title case or are they supposed to be sentence cased? Or does it vary? It just seems somewhat inconsistent, unless I'm misunderstanding. I was wondering this when I did the titles on the pages too